packages/@ama-sdk/core/package.json

```diff
@@ -47,7 +47,6 @@
     "prepare:publish": "prepare-publish ./dist"
   },
   "dependencies": {
-    "@o3r/schematics": "workspace:^",
```

Comment:
is it added by the `shell` generator?
Otherwise the `create` generation will fail.

Reply:
let me check